UI/Web/src/app/book-reader/_components/book-reader/book-reader.component.ts

```diff
@@ -1545,7 +1548,39 @@ export class BookReaderComponent implements OnInit, AfterViewInit, OnDestroy {
     return side === 'right' ? 'highlight-2' : 'highlight';
   }
 
+  maybeChangeCursor(event: MouseEvent) {
+    if (this.clickToPaginate) {
+      if (event.clientX <= window.innerWidth * 0.2 || event.clientX >= window.innerWidth * 0.8) {
+        this.pointer = true;
```

Comment:
You need to toggle change detection here. It would be more ideal for this to be an observable that maps to a book from mouse move. This way we can denounce.

Reply:
Got it. I'll take a look

Reply:
I tried debouncing, but there was noticeable delays in changing the cursor. I decided on a different approach, still with Observables. See https://github.com/Kareadita/Kavita/pull/2061/files#diff-b77cd1f8004a52c328b463987f49e3ef7910992e272358a539b654f891960d33R480-R487